### DISCUSSION TITLE
Winning a head to head feeds the winner

### DISCUSSION THREAD

**Discussion by BoldBigflank** [BattlesnakeOfficial/feedback#182] [General]
What might be an interesting strategy is to hunt and eat snakes, which could be rewarded by setting the winner's health to max.

**Comment by mawilson** [comment]
I like this idea! Definitely adds an interesting wrinkle, & I think it's logical.

**Comment by bvanvugt** [comment]
This seems pretty minimal and fun. I suspect the impact is pretty small, just encouraging head-to-head eliminations in a very small percentage of situations, yeah?
